user/security-in-qubes/split-gpg.md

```diff
@@ -77,6 +77,12 @@ Start with creating a dedicated AppVM for storing your keys (the GPG backend dom
 It is recommended that this domain be network disconnected (set its netvm to `none`) and only used for this one purpose.
 In later examples this AppVM is named `work-gpg`, but of course it might have any other name.
 
+You may edit the qrexec policy file for split GPG in order to tell Qubes your default gpg vm (qrexec prompts will appear with the gpg vm preselected as the target, instead of the user needing to type a name in manually). To do this, append `,default_target=<vmname>` to `ask` in `/etc/qubes-rpc/policy/qubes.Gpg`. For the examples given on this page:
+
+    @anyvm  @anyvm  ask,default_target=work-gpg
```

Comment:
I think both policy recommendations should be close to each other. Especially as the other one (see "Qubes 4.0 Specifics" section) generally is less prone to user errors. Also, since R3.2 is not supported anymore, "Qubes 4.0 Specifics" header is no longer necessary, as this part now applies to every supported Qubes version.